client/views/actions/away.tpl

```diff
@@ -0,0 +1,7 @@
+<span role="button" class="user {{colorClass from}}" data-name="{{from}}">{{mode}}{{from}}</span>
+{{#if text}}
+went away
+<i class="away-message">({{{parse text}}})</i>
+{{else}}
+came back
+{{/if}}
```

Comment:
`/nick` shows present tense (`x is now known as y`) so we could just go with `x is away` / `x is back` for consistency.

Also, I believe on The Lounge `/away` with no reason sets an away message of ` ` (one whitespace) so I think the message should now show up at all if trimmed `text` is empty.